### ISSUE TITLE
Training stopped

### ISSUE THREAD

**Issue by ThisIsForReview** [vislearn/FFF#3]
May someone have a look at the following errors?

lightning.fabric.utilities.exceptions.MisconfigurationException: `ModelCheckpoint(monitor='validation/nll_dec')` could not find the monitored key in the returned metrics: ['weights/nll', 'weights/noisy_reconstruction', 'training/z 1D-Wasserstein-1', 'training/z std', 'training/loss', 'training/nll', 'training/reconstruction', 'training/noisy_reconstruction', 'lr-Adam', 'validation/z 1D-Wasserstein-1', 'validation/z std', 'validation/loss', 'validation/nll', 'validation/vol_change_encoder', 'validation/vol_change_decoder', 'validation/nll_encoder', 'validation/nll_decoder', 'validation/reconstruction', 'validation/noisy_reconstruction', 'validation/z_reconstruction', 'validation/z_reconstruction_encoder', 'validation/z_sample_reconstruction', 'validation/x_sample_reconstruction', 'validation/shuffled_reconstruction', 'epoch', 'step']. HINT: Did you call `log('validation/nll_dec', value)` in the `LightningModule`?
Epoch 1: 100%|██████████| 391/391 [01:55<00:00,  3.38it/s, v_num=1, training/loss=0.592, validation/loss=12.60]

This is from the example 
python -m lightning_trainable.launcher.fit configs/fff/dw4.yaml --name '{data_set[name]}'

with the following revised dw4.yaml:

model: fff.FreeFormFlow

data_set:
  name: dw4
  root: data
# n_features: 0

lr_scheduler: "onecyclelr"

loss_weights:
  nll: 1
  noisy_reconstruction: 10

models:
  - name: fff.model.ENGNN
    n_dims: 2
    latent_dim: data
    n_features: 0
    n_layers: 1

model_checkpoint:
  monitor: validation/nll_dec
  save_last: True
  every_n_epochs: 2
  save_top_k: 1

latent_distribution:
  name: position-feature-prior
  n_dim: 2
  n_one_hot: 0
  n_charges: 0

batch_size: 256
optimizer:
  name: adam
  lr: 0.001
gradient_clip: 1.0
max_epochs: 50

accelerator: "cpu"
num_workers: 0

Thank you.

JB

**Comment by fdraxler** [comment]
Thanks for your question, we modified the name of the monitored key upon publishing the repo. I have just corrected the training configuration in 57ad0fa1514c449353528d5043570fca22e19ae0
